marlowe-dashboard-client/src/MainFrame/Types.purs

```diff
@@ -16,29 +9,27 @@ module MainFrame.Types
 
 import Prelude
 import Analytics (class IsEvent, defaultEvent, toEvent)
-import Contract.Types as Contract
 import Data.Either (Either)
 import Data.Generic.Rep (class Generic)
 import Data.Maybe (Maybe(..))
-import Marlowe.Semantics (PubKey)
+import Play.Types (Action, State) as Play
 import Plutus.PAB.Webserver.Types (StreamToClient, StreamToServer)
-import Template.Types (ContractSetupScreen, Template)
-import Template.Types (Action, State) as Template
-import WalletData.Types (WalletDetails, WalletLibrary, WalletNicknameKey)
+import Pickup.Types (Action, State) as Pickup
+import Template.Types (Template)
+import WalletData.Types (WalletLibrary, WalletNicknameKey)
 import Web.Socket.Event.CloseEvent (CloseEvent, reason) as WS
 import WebSocket.Support (FromSocket) as WS
 
 -- Apart from the wallet library (which you need in both cases), the app exists
--- in one of two distinct states: the "pickup" state for when you have no
--- wallet, and all you can do is pick one up or generate a new one; and the
--- "wallet" state for when you have picked up a wallet, and can do all
--- of the things.
+-- in one of two distinct states: the "pickup" state for when you have no wallet,
+-- and all you can do is pick one up or generate a new one; and the "play" state
+-- for when you have picked up a wallet, and can do all of the things.
```

Comment:
play is probably not the best word, but I can't figure out a better one

Reply:
My thoughts exactly. 🤷‍♂️ It'll have to do for now. It's better than "wallet", and anyway "wallet" led to a module naming clash.

Reply:
we have a new document for this that Simon made!